app/src/main/java/org/oppia/android/app/databinding/DrawableBindingAdapters.java

```diff
@@ -36,4 +36,14 @@ public static void setTopBackgroundDrawable(@NonNull View view, @ColorInt int co
     GradientDrawable background = (GradientDrawable) view.getBackground();
     background.setColor(0xff000000 | colorRgb);
   }
+
+  /** Used to set a rounded-rect background drawable with a data-bound color. */
+  @BindingAdapter("largeRoundedRectDrawableWithColor")
+  public static void setLargerBorderRadiusBackgroundDrawable(
+          @NonNull View view, @ColorInt int colorRgb) {
+    view.setBackgroundResource(R.drawable.rounded_rect_large_sync_status_chip);
```

Comment:
Follow 2-tab spacing + formatting suggestion:

```suggestion
  public static void setLargerBorderRadiusBackgroundDrawable(
    @NonNull View view, 
    @ColorInt int colorRgb
  ) {
    view.setBackgroundResource(R.drawable.rounded_rect_large_sync_status_chip);
```

Reply:
updated

Reply:
Please double check and update the indentation of the arguments to align with the standard formatting.

Reply:
updated

Reply:
Removing extra 2 tabs resulted in failing checkstyle check.